README.md

```diff
@@ -16,22 +16,22 @@ In the QA team, we welcome highly skilled and proactive people who want to help
 the best tests and coach the team to meet their best in terms of quality.
 
 ## Challenge Description
-Please, create a test plan with the test cases you think should be covered in this page. You don't need to automate all of them, just the ones that you consider more important oor critical in this scenario.
+Please, create a test plan with the test cases you think should be covered the login and the inventory view of the page. You don't need to automate all of them, just the ones that you consider more important or critical in this scenario. The other ones can left mentioned on a testplan doc, or commented in the code. You decide how much want to share about your automation skills!
```

Comment:
Should be : "should be covered _for_ the login and the inventory view of the page."